src/cookbook/editable-svg-icons.md

```diff
@@ -1,26 +1,26 @@
-# Editable SVG Icon Systems
+# Sistema de íconos SVG editables
 
-## Base Example
+## Ejemplo base
 
-There are many ways to create an SVG Icon System, but one method that takes advantage of Vue's capabilities is to create editable inline icons as components. Some of the advantages of this way of working is:
+Existen muchas maneras de crear un sistema de íconos SVG, pero un método que aprovecha las capacidades de Vue, es crear íconos _inline_ editables como componentes. Algunas de las ventajas de esta forma de trabajar son:
 
-- They are easy to edit on the fly
-- They are animatable
-- You can use standard props and defaults to keep them to a typical size or alter them if you need to
-- They are inline, so no HTTP requests are necessary
-- They can be made accessible dynamically
+- Son fáciles de editar en tiempo real
+- Son animables
+- Puede usar _props_ y valores _default_, para mantener un tamaño común o moficarlo si es necesario
```

Comment:
I think we can translate `default`, what do you think?
```suggestion
- Puede usar _props_ y valores por defecto, para mantener un tamaño común o moficarlo si es necesario
```

Reply:
Sounds good! Thanks @Tintef